.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "RTTR/languages"]
 	url = https://github.com/Return-To-The-Roots/languages.git
 	path = RTTR/languages
+[submodule "contrib/kaguya"]
+	path = contrib/kaguya
+	url = https://github.com/satoren/kaguya.git
```

Comment:
I thought about that. But I did not see an advantage (for now). The guy maintaining it is pretty active and this way we are better linked to the upstream. We maintain the other repos, hence they are in our scope, but I don't want/need us to maintain kaguya ;)